### PR TITLE
Completed css-flexbox exercise. -- Joshua Preciado

### PR DESCRIPTION
**Pull request by joshuapreciado**
PIKA

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    body {
```
Comment (BrettAlbright) [inline css-flexbox/align-items.html]: this should be on the card-deck

File: css-flexbox/align-items.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      display: flex;
+      flex-basis: calc(100% / 3);
```
Comment (BrettAlbright) [inline css-flexbox/align-items.html]: the `flex-basis` should be in a selector for the `card-wrapper`

File: css-flexbox/display-flex.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    div {
```
Comment (BrettAlbright) [inline css-flexbox/display-flex.html]: this should be for the `card-deck`

File: css-flexbox/flex-wrap.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    div {
```
Comment (BrettAlbright) [inline css-flexbox/flex-wrap.html]: should be the card-deck selector

File: css-flexbox/flex-wrap.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```
Comment (BrettAlbright) [inline css-flexbox/flex-wrap.html]: This should be in the `card-wrapper` selector

File: css-flexbox/justify-content.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    div {
```
Comment (BrettAlbright) [inline css-flexbox/justify-content.html]: this should be the `card-deck` selector

File: css-flexbox/justify-content.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+      display: flex;
+      flex-basis: 25%;
```
Comment (BrettAlbright) [inline css-flexbox/justify-content.html:16]: this should be in the wrapper for the  card

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
good initial submission, some updates are needed for the selectors!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny updates, `card-wrapper` is a class, so make sure to update the selector, and the flex-basis for the `justify-content` page

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**